.travis.yml

```diff
@@ -31,6 +31,14 @@ jobs:
     - &test-pyinstaller
       stage: Test - PyInstaller
       python: 3.7
+      before_install:
+        - >
+          [ -f tests/functional/data/appimage/appimagetool-x86_64.AppImage ] || (
+              wget \
+                https://github.com/AppImage/AppImageKit/releases/download/12/appimagetool-x86_64.AppImage \
+                -O tests/functional/data/appimage/appimagetool-x86_64.AppImage \
+                && chmod a+x tests/functional/data/appimage/appimagetool-x86_64.AppImage
```

Comment:
Putting this file into the source will *always* mark the version as "dirty".

What about downloading it to $HOME? (And in the test-case use `pathlib.Path.home() / "appimagetool-x86_64.AppImage"`as filename)